ui/app/pages/connected-accounts/index.scss

```
@@ -0,0 +1,8 @@
+.connected-accounts {
+  &__footer {
+    a, a:hover {
+      color: #037DD6;
+      cursor: pointer;
+    }
```

Nit: The designs have this as 14px at the moment, rather than 16px as it is currently.

```suggestion
      14px;
    }
```